x/fees/spec/02_state.md

```diff
@@ -0,0 +1,41 @@
+<!--
+order: 2
+-->
+
+# State
+
+## State Objects
+
+The `x/fees` module keeps the following objects in state:
+
+| State Object      | Description                | Key                                    | Value                        | Store |
+|-------------------|----------------------------|----------------------------------------|------------------------------|-------|
```

Comment:
Looks very good.
Worth mentioning to enforce the left alignment of table content with  :--